src/stylesheets/components/_accordions.scss

```diff
@@ -1,6 +1,7 @@
 // Variables
 
-$accordion-border: units(0.5) solid color('base-lighter');
+$accordion-border: units(0.5) solid color('base-lightest');
+$accordion-font-family: 'sans' !default;
```

Comment:
is this a pattern that we plan to do on all components?

Reply:
I'm not really sure yet — but because font size and family are fundamentally coupled in v2 I've been playing with different techniques to make sure each component is accessing the proper value. I think after I muddle through typography I'll have a better attempt at guidance here